src/vm/types/range_check_instance_def.zig

```diff
@@ -12,15 +14,36 @@ pub const RangeCheckInstanceDef = struct {
     /// For example, n_parts=8 defines the range [0, 2^128).
     n_parts: u32,
 
-    /// Number of units per builtin
-    pub fn rangeCheckUnitsPerBuiltin(self: *Self) u32 {
-        return self.n_parts;
-    }
-
     pub fn default() Self {
         return .{
             .ratio = 8,
             .n_parts = 8,
         };
     }
+
+    /// Creates a new instance of `RangeCheckInstanceDef` with the specified ratio and n_parts representation.
+    ///
+    /// # Parameters
+    ///
+    /// - `ratio`: An optional 32-bit integer representing the ratio for the Range check instance.
+    /// - `n_parts`: An 32-bit integer representing number of parts.
+    ///
+    /// # Returns
+    ///
+    /// A new `RangeCheckInstanceDef` instance with the specified parameters.
+    pub fn init(ratio: ?u32, n_parts: u32) Self {
```

Comment:
I would maybe call this one: `from` or something like that since it involves this time parameters.